application/classes/Ushahidi/Formatter/Post/CSV.php

```diff
@@ -81,15 +83,37 @@ protected function generateCSVRecords($records)
 
 	private function getValueFromRecord($record, $keyParam){
 		$return = '';
-		$keySet = explode('.', $keyParam); //contains key + index of the key, if any
+		$keySet = explode('.', $keyParam); //contains key + index of the key
+		if (count($keySet) == 1) {
+			preg_match('/([a-zA-Z]+)([0-9]+)\(([0-9]+)\)/',$keyParam, $match);
+			if ($match) {
+				$keySet[0] = $match[1].$match[2];
+				$keySet[1] = $match[1];
+				$keySet[2] = $match[3];
+			}
+		}
+
 		$headingKey = $keySet[0];
 		$key = isset($keySet[1]) ? $keySet[1] : null;
-		$recordValue = isset ($record['attributes']) && isset($record['attributes'][$headingKey])? $record['values']: $record;
-		if($key === 'lat' || $key === 'lon'){
+		$recordAttributes = isset($record['attributes'][$headingKey]) ? $record['attributes'][$headingKey] : null;
+		$recordValue = isset ($record['attributes']) && $recordAttributes? $record['values']: $record;
+		$isDateField = $recordAttributes['input'] === 'date' && $recordAttributes['type'] === 'datetime';
+
+		if ($isDateField) {
```

Comment:
not sure if the right place..  but I think there's not much to this. Basically, it just needs to be switched to y-m-d